api/server/capture.go

```diff
@@ -118,6 +118,12 @@ func (c *CaptureAPIHandler) Create(r types.Resource) error {
 			}
 		}
 	}
+	if capture.SamplingRate == 0 {
+		capture.SamplingRate = 1
+	}
+	if capture.PollingInterval == 0 {
```

Comment:
we can check less than 10. if 10 is the minimum value.

Reply:
Range of PollingInterval is 0 to 3600 seconds , default for SFlow is 2 seconds but with average 10 seconds will get better results! :)